src/taker/config.rs

```diff
@@ -44,17 +20,83 @@ pub struct TakerConfig {
 
 impl Default for TakerConfig {
     fn default() -> Self {
-        Self {
-            refund_locktime: REFUND_LOCKTIME,
-            refund_locktime_step: REFUND_LOCKTIME_STEP,
-            first_connect_attempts: FIRST_CONNECT_ATTEMPTS,
-            first_connect_sleep_delay_sec: FIRST_CONNECT_SLEEP_DELAY_SEC,
-            first_connect_attempt_timeout_sec: FIRST_CONNECT_ATTEMPT_TIMEOUT_SEC,
-            reconnect_attempts: RECONNECT_ATTEMPTS,
-            reconnect_short_sleep_delay: RECONNECT_SHORT_SLEEP_DELAY_SEC,
-            reconnect_long_sleep_delay: RECONNECT_LONG_SLEEP_DELAY_SEC,
-            short_long_sleep_delay_transition: SHORT_LONG_SLEEP_DELAY_TRANSITION,
-            reconnect_attempt_timeout_sec: RECONNECT_ATTEMPT_TIMEOUT_SEC,
+        let sections = parse_toml("config.toml").unwrap();
```

Comment:
It could be problematic to have the file path hardcoded. This should be an arg of the `parse` function for taker/maker config. Better to use an `Option<PathBuff>`, with a default hardcoded path.